infrastructure/code/code.go

```diff
@@ -377,8 +377,16 @@ func (sc *Scanner) createBundle(ctx context.Context,
 		rootPath:      rootPath,
 		errorReporter: sc.errorReporter,
 		limitToFiles:  limitToFiles,
-		learnService:  sc.learnService,
 		notifier:      sc.notifier,
+		issueEnhancer: IssueEnhancer{
+			SnykCode:      sc.BundleUploader.SnykCode,
+			instrumentor:  sc.BundleUploader.instrumentor,
+			errorReporter: sc.errorReporter,
+			learnService:  sc.learnService,
+			notifier:      sc.notifier,
+			requestId:     requestId,
```

Comment:
RequestID is for one Snyk Code scan, issueEnhancer potentially for a session. How do we treat updates to the requestID, or asked differently - can we make the requestID transient?